src/test/java/com/flightstats/hub/model/ChannelConfigTest.java

```diff
@@ -137,13 +133,13 @@ public void testCopy() throws IOException {
                 .mutableTime(TimeUtil.now())
                 .allowZeroBytes(false)
                 .build();
-        assertTrue(config.equals(config.toBuilder().build()));
 
-        assertTrue(config.equals(ChannelConfig.createFromJson(config.toJson())));
+        assertEquals(config, config.toBuilder().build());
```

Comment:
...I don't know what `.toBuilder()` does and I'm a little bit afraid to look. I know you didn't introduce it, so don't feel obliged to answer if you don't know.

Reply:
toBuilder() is full of awesomeness. It creates an exact close of the original object, but returns a builder. Great for  "I need a clone, but with this one field changed" situations.

That said...no idea why you'd to a toBuilder() and then immediately .build().

Reply:
Ah, neat! I usually use the lombok annotation that lets you do `withAttribute()` chaining to accomplish something similar. ...As you pointed out, I agree that the main concern was the immediate `build()` that happens afterwards.

Reply:
I am guessing the purpose of this was to test that the (all hand-rolled) hash and equals and builder and toBuilder worked in unison to create an exact copy. Which IIRC is implicitly tested on the next line as createFromJson uses that method under the hood.

Reply:
Ah, I guess it'd make sense to leave it there at least temporarily to make sure re-lombokifying things works out if you decide to make that change.